allium/static/css/allium-base.css

```
@@ -0,0 +1,613 @@
+        padding-left: 15px;
+        margin-right: auto;
+        margin-left: auto;
+    }
+}
+
+/* Fix two-column layout overflow in contact/operator pages */
+.row > .col-md-7,
+.row > .col-md-5 {
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)